Sprint-2/implement/contains.test.js

```diff
@@ -33,3 +32,18 @@ test.todo("contains on empty object returns false");
 // Given invalid parameters like an array
 // When passed to contains
 // Then it should return false or throw an error
+
+describe("contains", () => {
+  test("When passed an object and a property name,it should return true if the object contains the property, false otherwise", () => {
+    expect(contains({a: 1, b: 2}, "a" )).toBe(true);
+  });
+   test("Given an empty object,it should return false", () => {
+     expect(contains({})).toEqual(false);
+   });
+   test("When passed to contains with a non-existent property name,it should return false", () => {
+     expect(contains({a:1, b:2}, "c")).toEqual(false);
+   });
+   test("Given invalid parameters like an array,it should return false or throw an error", () => {
+    expect(contains(["a", "b", "c"], "c")).toEqual(false)
+   })
```

Comment:
What do you expect from the following function calls, and did your function returns the expected value?
```
contains(["1", "2", "3"], "2")
contains(undefined, undefined)
contains(null, "a")
```